src/app/onboarding/(components)/onboarding-background.tsx

```
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const OnboardingBackground = () => {
+  return (
+    <div className="absolute inset-0 bg-white opacity-10">
+      <svg className="size-full" xmlns="http://www.w3.org/2000/svg">
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)